lib/asset/baseAsset.js

```diff
@@ -242,6 +243,11 @@ class BaseAsset {
         }
 
         hatch._assets.push(this);
+        if (this._has_failed) {
+            hatch._failed_assets.push(this);
```

Comment:
Is it OK that we are putting this asset both in the regular and the failed lists?

Reply:
Yes, the successful assets are calculated by filtering failed from regular, and then the % of failed is calculated as successful / regular .